### PR TITLE
superscrollbar improvements Requesting a pull to mozilla:master from k88hudson:t2392

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @brett-etherworks received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/e08178d09234dea/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/e08178d09234dea/output.txt

Total script time: 1.56 mins

Server is running on http://pr1045.butyr.org/templates/basic/
Run the Test Suite: http://pr1045.butyr.org/test/

File: css/super-scrollbar.less [hunk 1]
```diff
@@ -58,31 +57,82 @@
...
   display: block;
-  height: 14px;
-  margin-top: -7px;
+  height: 18px;
+  margin-top: -9px;
   position: absolute;
   top: 50%;
-  width: 4px;
+  width: 6px;
+  border: 1px solid #2A7B60;
```
Comment (secretrobotron) [inline css/super-scrollbar.less]: You'll probably want to make a const for these, right?

File: css/super-scrollbar.less [hunk 2]
```diff
@@ -58,31 +57,82 @@
...
+    opacity: 0.5;
+  }
+  &:before {
+    content: "";
+    position: absolute;
+    top: 2px;
+    bottom: 2px;
+    left: 2px;
+    width: 2px;
+    background: #2A7B60;
```
Comment (secretrobotron) [inline css/super-scrollbar.less]: re https://github.com/mozilla/butter/pull/1045/files#L0R91

File: css/super-scrollbar.less [hunk 3]
```diff
@@ -58,31 +57,82 @@
...
+    width: 100%;
+    height: 100%;
+    position: absolute;
+    top: -2px;
+    bottom: -2px;
+    left: 0;
+    display: block;
+    border: 4px solid #FFFFFF;
+    z-index: 1000;
+    box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.2) inset;
```
Comment (secretrobotron) [inline css/super-scrollbar.less]: Spacing for `rgba`.

File: css/super-scrollbar.less [hunk 4]
```diff
@@ -58,31 +57,82 @@
...
+  }
 }
 
 .butter-super-scrollbar-trackevent {
-  background: none repeat scroll 0 0 #999999;
+  background: rgba( 0, 0, 0, 0.4 );
   height: 3px;
   position: absolute;
+  z-index: 1000;
```
Comment (secretrobotron) [inline css/super-scrollbar.less]: What is `1000`? Should be based on something else?

File: css/super-scrollbar.less [hunk 5]
```diff
@@ -58,31 +57,82 @@
...
 }
 
 .butter-super-scrollbar-handle {
-  background: none repeat scroll 0 0 #41BB92;
+  background: #41BB92;
```
Comment (secretrobotron) [inline css/super-scrollbar.less]: re https://github.com/mozilla/butter/issues/1045

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @dseif received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/a4082bb2ab4e27d/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @dseif received. Current queue size: 2

Live output at: http://botio.butyr.org:8000/4bb8443b748d28b/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @dseif received. Current queue size: 3

Live output at: http://botio.butyr.org:8000/9a75176767baff0/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/a4082bb2ab4e27d/output.txt

Total script time: 3.00 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/4bb8443b748d28b/output.txt

Total script time: 3.00 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/9a75176767baff0/output.txt

Total script time: 3.00 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @dseif received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/2f4ab29a190e79d/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/2f4ab29a190e79d/output.txt

Total script time: 1.43 mins

Server is running on http://pr1045.butyr.org/templates/basic/
Run the Test Suite: http://pr1045.butyr.org/test/

File: src/timeline/super-scrollbar.js [hunk 6]
```diff
@@ -45,14 +47,23 @@ define( [ "util/lang", "text!layouts/super-scrollbar.html" ],
...
         _this = this;
 
-    var onViewMouseUp, onViewMouseDown, onViewMouseMove,
+    var checkMinSize, onViewMouseUp, onViewMouseDown, onViewMouseMove,
         onLeftMouseUp, onLeftMouseDown, onLeftMouseMove,
         onRightMouseUp, onRightMouseDown, onRightMouseMove,
         onElementMouseUp, onElementMouseDown, onElementMouseMove,
         updateView;
 
+    checkMinSize = function () {
```
Comment (secretrobotron) [inline src/timeline/super-scrollbar.js]: Spacing after `function`

Comment (k88hudson) [inline src/timeline/super-scrollbar.js]: why do I suck so much at whitespace

**Pull request merged**